core/server/common/src/main/java/alluxio/master/journal/Journal.java

```diff
@@ -11,19 +11,19 @@
 
 package alluxio.master.journal;
 
+import alluxio.master.journal.options.JournalReaderCreateOptions;
```

Comment:
hmm, it is a little different here. They are the options used to create the JournalReader/JournalWriter. In other words, they are options used to call the constructors. Maybe they can be called JournalReaderOptions and JournalWriterOptions. I am fine with that.